suite-native/assets/src/components/Assets.tsx

```diff
@@ -67,18 +67,24 @@ export const Assets = () => {
                     ))}
                 </VStack>
             </Card>
-            <BottomSheet
```

Comment:
Something is really messed up in this file, you should not delete BottomSheet nor add extra buttons :D Maybe this happened because of conflict because these button was moved to separated component.

Reply:
Should be fixed now.